planty/chats/utils.py

```diff
@@ -0,0 +1,22 @@
+
+from .models import Message, Chat
+from django.db.models import Q
+
+
+def return_list_last_messages_in_chat(request):
+    all_users_chats = Chat.objects.filter(Q(buyer=request.user) | Q(
+        post__user=request.user))
+
+    list_last_messages_in_chat = []
+    for chat in all_users_chats:
```

Comment:
I think it should be possible to avoid the for loop in python and use django orm to get all the messages (it would be faster and more concise code). But we can leave this as it is for now